config.py

```
@@ -1,18 +1,21 @@
 import os
 BASE_DIR = os.path.abspath(os.path.dirname(__file__))
 
-DEBUG = True
+DEBUG = os.environ.get('DEBUG', True)
 
-DATABASE_URI = 'sqlite:///' + os.path.join(BASE_DIR, 'audit.db')
+DATABASE_URI = os.environ.get('DATABASE_URI', 'sqlite:///' + os.path.join(BASE_DIR, 'audit.db'))
 # DATABASE_URI = 'postgresql://localhost/cf_audit'
-MAX_CONTENT_LENGTH = 16*1024*1024
+MAX_CONTENT_LENGTH = int(os.environ.get('MAX_CONTENT_LENGTH', 16*2014*1024))
 
-ADMINS = set([290271])  # Zverik
+ADMINS = set(os.environ.get('ADMINS', '').split(','))
+if not ADMINS:
```

I think you can remove this, otherwise people would get myself as an admin. Maybe throw an exception?

---

I'll see what I can do and submit a changed version, best combined with the `OAUTH_KEY` and similar settings. Might need further refinement.